specification/synapse/data-plane/Microsoft.Synapse/preview/2021-07-01-preview/symsSync.json

```diff
@@ -0,0 +1,326 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2021-07-01-preview",
+    "title": "ArtifactsClient"
+  },
+  "x-ms-parameterized-host": {
+    "hostTemplate": "{endpoint}",
+    "useSchemePrefix": false,
+    "parameters": [
+      {
+        "$ref": "#/parameters/Endpoint"
+      }
+    ]
+  },
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/synapsemetastore/register": {
+      "put": {
+        "tags": [
+          "synapsemetastore"
+        ],
+        "operationId": "SynapseMetastore_Register",
+        "description": "Register files in Syms",
+        "x-ms-examples": {
+          "Register files in Syms": {
+            "$ref": "./examples/SynapseMetastore_Register.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "in": "body",
+            "name": "registerBody",
+            "required": true,
+            "description": "The register body",
+            "schema": {
+              "$ref": "#/definitions/registerObject"
+            }
+          }
+        ],
+        "responses": {
+          "201": {
+            "description": "Metadata Request Accepted Response.",
+            "schema": {
+              "$ref": "#/definitions/metadataRequestAcceptedResponse"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ErrorContract"
+            }
+          }
+        }
+      }
+    },
+    "/synapsemetastore/update": {
+      "put": {
+        "tags": [
+          "synapsemetastore"
+        ],
+        "operationId": "SynapseMetastore_Update",
```

Comment:
>SynapseMetastore_Update [](http://example.com/codeflow?start=24&length=23)

IIRC, we got a suggestion from API review board that we should avoid using marking name (here I am referring to Synapse) in the operation id (in SDK it will be the client name).

@lmazuel what do you think?

Reply:
done , removed the prefix synapse